lms/templates/labster/course_license.html

```diff
@@ -29,7 +29,7 @@
     }
     .label {color: #646464}
     .messages {margin: 10px 0 20px}
-    .messages .error {color: red}
+    .messages .error, .messages .success {color: red}
```

Comment:
@lukyfebrianto why success message is red? Shouldn't it be green?

Reply:
I see the picture on the ticket is red @polesye,  should I change to green ?
CMIIW

Reply:
yes, please. I think it was a quick mockup

Reply:
updated